packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -231,9 +242,15 @@ class Tooltip extends StatefulWidget {
 
   /// Whether the tooltip defaults to being displayed below the widget.
   ///
-  /// Defaults to true. If there is insufficient space to display the tooltip in
+  /// If there is insufficient space to display the tooltip in
   /// the preferred direction, the tooltip will be displayed in the opposite
   /// direction.
+  ///
+  /// If this property is null, then [TooltipThemeData.preferBelow] is used.
+  /// If that is also null, the default value is true.
+  ///
+  /// Applying [TooltipThemeData.preferBelow]: `false` for your entire app
+  /// is recommended to avoid having a finger or cursor hide a tooltip.
```

Comment:
Hmm, it actually seems a bit nontrivial to work out from this, in the context of the existing docs, *how* one can go about following this advice.

From [TooltipThemeData.preferBelow] one can naturally follow a link to [TooltipThemeData]. But then it's not clear from there what one does with a [TooltipThemeData] to make it take effect.

Well, I guess from that one learns about [TooltipTheme] widgets, and can use one of those. But I think the idiomatic thing, at least when applying it to the whole app, is probably to wrap it up in a [ThemeData] and use either a [Theme] widget or [MaterialApp.theme].

I guess maybe the simplest solution is if I send my own small PR adding a couple of appropriate cross-references there. This advice is still helpful, in that people can use `TooltipTheme`, or look at the examples above, or just already have a mental picture of the Material "theme" system and so already know where to slot in this added fact.

Reply:
Sent #137316 with those other cross-references, and some other tweaks I found I wanted to make once I started editing. (All still orthogonal to this PR, I think.)